models.py

```diff
@@ -892,19 +954,145 @@ class FeatureForm(forms.Form):
+      help_text='One URL per line.')
+
+  intent_to_implement_url = forms.URLField(required=False, label='Intent to Implement link',
+      help_text='Link to the "Intent to Implement" discussion thread.')
+
+  origin_trial_feedback_url = forms.URLField(required=False, label='Origin Trial feedback summary',
+      help_text='If your feature was available as an Origin Trial, link to a summary of usage and developer feedback. If not, leave this empty.')
+
+  doc_links = forms.CharField(label='Doc link(s)', required=False, max_length=500,
+      widget=forms.Textarea(attrs={'rows': 4, 'cols': 50, 'placeholder': 'Links to design doc(s), if and when available. [This is not required to send out an intent to implement. Please update the intent thread with the design doc when ready]. An explainer and/or design doc is sufficient to start this process.\n\n[Note: Please include links and data, where possible, to support any claims.]', 'maxlength': 500}),
+      help_text='One URL per line')
+
+  standardization = forms.ChoiceField(
+      label='Standardization', choices=STANDARDIZATION.items(),
+      initial=EDITORS_DRAFT,
+      help_text=("The standardization status of the API. In bodies that don't "
+                 "use this nomenclature, use the closest equivalent."))
+
+  spec_link = forms.URLField(required=False, label='Spec link',
+                             help_text="Link to spec, if and when available. Please update the chromestatus.com entry and the intent thread(s) with the spec link when available.")
+
+  tag_review = forms.CharField(label='TAG Review', required=True,
+      widget=forms.Textarea(attrs={'rows': 2, 'cols': 50, 'placeholder': 'Link(s) to TAG review(s), or explanation why this is not needed.', 'maxlength': 500}),
+      help_text='One URL per line, optionally followed by a short title, or rationale for omitting TAG review.')
+
+  interop_compat_risks = forms.CharField(label='Interoperability and Compatibility Risks', required=True, max_length=500,
+      widget=forms.Textarea(attrs={'cols': 50, 'maxlength': 500}),
+      help_text='Describe the degree of <a target="_blank" href="https://sites.google.com/a/chromium.org/dev/blink?pli=1#TOC-Policy-for-shipping-and-removing-web-platform-API-features">interoperability risk</a>. For a new feature, the main risk is that it fails to become an interoperable part of the web platform if other browsers do not implement it. For a removal, please review our <a target="_blank" href="https://docs.google.com/document/d/1RC-pBBvsazYfCNNUSkPqAVpSpNJ96U8trhNkfV0v9fk/edit">principles of web compatibility</a>.')
```

Comment:
Not sure this will work. Do the links render properly in the textarea?

Reply:
Yep, the help text ends up below the actual textarea (as opposed to the "placeholder" text which can not contain markup AFAIK).